addons/account/migrations/12.0.1.1/openupgrade_analysis.txt

```diff
@@ -1,10 +1,11 @@
 ---Fields in module 'account'---
-account      / account.account          / internal_group (selection)    : NEW selection_keys: function
+account      / account.account          / internal_group (selection)    : NEW isrelated: related, stored: stored, selection_keys: function
+account      / account.account.tag      / color (integer)               : without default
```

Comment:
This text disappears now, isn't it?

Reply:
yes, don't merge this yet, I need to redo the analysis

Reply:
Yeah, I was waiting for that :smiley: